src/outputs/parthenon_hdf5.cpp

```diff
@@ -202,16 +202,24 @@ struct VarInfo {
   int nx6;
   int nx5;
   int nx4;
+  int ndim; // 1-, 2-, or 3-D
+  MetadataFlag where;
```

Comment:
why not just also report `nx3`, `nx2`, `nx1`? One could imagine these might not match meshblock shape. (In fact, I explicitly want us to support where they sometimes don't. For example geometry in Phoebus.)

Reply:
Here I was using `where = Metadata::none` to account for non-meshblock-shaped variables (in fact I was specifically thinking of the Phoebus geometry :) ). The one issue with just getting all six `nx*` is that meshblock-tied and non-meshblock-tied variables have different `shape`s. For example the Phoebus geometry has some 4D `shape` for a 4D output grid whereas a cell-centered tensor in Phoebus will have a 2D shape and a 3D meshblock size for a 5D output grid. So at some point here we need to use `where` to make this distinction regardless.

Reply:
hmm ok not sure I fully understand what you mean. But I guess it'll make sense when you're finished tweaking.

Reply:
So it actually turned out to be necessary to store `nx1` `nx2` and `nx3` here to resolve the issue that showed up with axisymmetric geometry variables in phoebus :)